documentation/en/source/pages/migration/migration-spec/migration-source.rst

```diff
@@ -31,4 +31,5 @@ The valid keys to use within a source are:
 * ``ssh-port`` (optional): an integer providing the ssh port of the running system (default: 22).
 * ``user`` (mandatory): a string providing the user login to authenticate to the live system.
 * ``password`` (optional): a string providing the user password to authenticate to the live system.
-* ``exclude`` (optional): a list of non-native files or directories to exclude during the deep scan of the live system.
\ No newline at end of file
+* ``exclude`` (optional): a list of non-native files or directories to exclude during the deep scan of the live system.
+* ``overlay`` (optional): a boolean to include overlay (extra files) for the migration (default: true)
```

Comment:
shouldn't we keep coherency with `exclude` parameter and use `during the deep scan of the live system`?

Reply:
Need to discuss.
I proposed this based on scan documentation:
--exclude (optional): a list of directories or files to exclude during the deep scan
--overlay (optional): include overlay (extra files) for the given scan

Reply:
coherent with scan, forget my comment